src/test/nexusTestUtils.js

```diff
@@ -52,104 +52,167 @@ fluid.test.nexus.assertContainsComponent = function (componentRoot, parentPath,
     jqUnit.assertValue(parentPath + " component contains " + childName, parent[childName]);
 };
 
+/**
+ * Assert that the object toTest contains a set of keys with given values.
+ * Can also be understood as "assert that the object expected is a subset of the object toTest".
+ * @param {Object} expected the expected keys and values.
+ * @param {Object} actual the object to test.
+ */
+fluid.test.nexus.assertKeyValues = function (expected, actual) {
+    for (var key in expected) {
+        var value = expected[key];
+        jqUnit.assertDeepEq(value, actual[key]);
+    }
+};
+
+fluid.defaults("fluid.test.nexus.readDefaultsRequest", {
+    gradeNames: ["kettle.test.request.http"],
+    path: "/defaults/%gradeName",
+    port: "{configuration}.options.serverPort",
+    method: "GET",
+    termMap: {
+        gradeName: "fill in construction options"
+    }
+});
+
+fluid.defaults("fluid.test.nexus.writeDefaultsRequest", {
+    gradeNames: ["kettle.test.request.http"],
+    path: "/defaults/%gradeName",
```

Comment:
You could, in the current nerdish parlance, "DRY" these up by factoring gradeNames, path, port and termMap into a base grade